shell/platform/embedder/embedder.h

```diff
@@ -2434,6 +2434,26 @@ typedef struct {
   /// being registered on the framework side. The callback is invoked from
   /// a task posted to the platform thread.
   FlutterChannelUpdateCallback channel_update_callback;
+
+  /// The version of the app (e.g., 1.0.0). Used by Shorebird.
+  const char* app_version;
```

Comment:
It's not clear what the lifetime of these strings are.  When do they get freed?  How long are they expected to be valid for?

Reply:
Added comments explaining lifetime expectations